tests/unit/package/builder/test_package_builder_isolated.py

```diff
@@ -28,6 +28,27 @@ def test_verbose_isolated_build(initproj, mock_venv, cmd):
     assert "Writing example123-0.5{}setup.cfg".format(os.sep) in result.out, result.out
 
 
+def test_isolated_build_develop(initproj, cmd):
+    initproj(
+        "example123-0.5",
+        filedefs={
+            "tox.ini": """
+                    [tox]
+                    isolated_build = true
+                    [testenv]
+                    usedevelop = true
+                    """,
+            "pyproject.toml": """
+                    [build-system]
```

Comment:
How about another test without build-system? And +1 with build system but no build-backend...